modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/test/DDMStructureManagerTest.java

```diff
@@ -317,11 +318,16 @@ protected DDMStructure addStructure() throws Exception {
 			LocaleUtil.US, "Test Structure Description"
 		).build();
 
-		return _ddmStructureManager.addStructure(
+		DDMStructure ddmStructure = _ddmStructureManager.addStructure(
 			TestPropsValues.getUserId(), _group.getGroupId(), null,
 			_classNameId, StringUtil.randomString(), nameMap, descriptionMap,
 			createDDMForm(), StorageEngineManager.STORAGE_TYPE_DEFAULT,
 			DDMStructureManager.STRUCTURE_TYPE_DEFAULT, _serviceContext);
+
+		_ddmStructure = DDMStructureLocalServiceUtil.getStructure(
+			ddmStructure.getStructureId());
+
+		return ddmStructure;
```

Comment:
Hey @carolmariaabb, I believe this test class has room for optimization, therefore let's do the following:

- Keep the method `addStructure` as it was before your modification.
- Add a single call to the method `addStructure` as it's being done to [add a group](https://github.com/liferay/liferay-portal/blob/dedeb279e3dfed0f1d0074682df08b919ff4d5f9/modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/test/DDMStructureManagerTest.java#L76).
- Remove all calls to `addStructure` that are in the tests.

Can you please resend your pull observing the suggestions above?

Thanks.